Tests/Expected/Storyboards-iOS/default-context-all-customname.swift

```diff
@@ -1,5 +1,6 @@
 // Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
 
+// swiftlint:disable sorted_imports
```

Comment:
And even if we could, we'd need to add the imports UIKit or Cocoa to that list, and only then sort it.